README.md

```diff
@@ -38,16 +38,21 @@ To install:
 
     pip install qtconsole
 
-**Note:** Make sure that Qt is installed. Unfortunately, Qt cannot be
-installed using pip. The next section gives instructions on installing Qt.
+**Note:** Make sure that Qt is installed. The next section gives instructions on installing Qt.
 
 ### Installing Qt (if needed)
 We recommend installing PyQt with [conda](http://conda.pydata.org/docs):
 
     conda install pyqt
 
 or with a system package manager. For Windows, PyQt binary packages may be
-used.
+used:
+
+    pip install pyqt5
```

Comment:
Actually, this now works (or should work) on 64-bit Linux and Mac as well.

Reply:
I don't have a Mac nor a Linux VM, so I didn't dare writing about that. Maybe you can complete the update ?

Reply:
I think it does work, so I'd just remove the phrasing that says it's only for Windows.

Reply:
Thomas is right, this works for macOS and Linux as well.